Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -2066,12 +2066,20 @@ bool RenderElement::hasSelfPaintingLayer() const
 
 bool RenderElement::capturedInViewTransition() const
 {
-    if (!style().viewTransitionName())
+    if (!hasViewTransitionProperty())
         return false;
 
     return !!document().activeViewTransition();
 }
 
+bool RenderElement::hasViewTransitionProperty() const
```

Comment:
```suggestion
bool RenderElement::hasViewTransitionName() const
```

but also why the split?